conftest.py

```diff
@@ -2,6 +2,8 @@
 
 import pytest
 
+import borg.cache
```

Comment:
I'm not entirely sure why this was a `pytest_configure` before, but i'll add a commit moving that to the top without using `pytest_configure`